release/release.go

```diff
@@ -1108,17 +1113,42 @@ func uploadRelease(v version.Version) {
 	}
 }
 
-var linuxRepoVersionsStable = map[string]string{
-	"development": "4.0.0-15-gabcde123", // any non-rc pre-release version will send the package to the "development" repo
-	"testing":     "4.0.0-rc0",          // any rc version will send the package to the "testing" repo
-	"4.3":         "4.3.0",              // any 4.3 stable release version will send the package to the "4.3" repo
-	"4.4":         "4.4.0",              // any 4.4 stable release version will send the package to the "4.4" repo
-	"4.9":         "4.9.0",              // any 4.9 stable release version will send the package to the "4.9" repo
-	"5.0":         "5.0.0",              // any 5.0 stable release version will send the package to the "5.0" repo
+type VersionWithSigningInfo struct {
```

Comment:
```suggestion
type LinuxRepo struct {
    name string
```
a couple suggestions here:
- this struct really represents a linux repo, not a version, so let's call it `linuxRepo`
- since we have a struct now, I'd suggest that we just include the repo name as another field. Then `linuxRepoVersions{Stable,Unstable}` can just be `[]linuxRepo` instead of maps

Reply:
Make sense. made the change.